duckduckgo/iOS
# Feature/whitelisting

Reviewer: Mia
Asana: https://app.asana.com/0/0/427251562039378/f
CC:

**Description**:

Add support for whitelisting.

**Steps to test this PR**:

Test 1:
1. Visit a tracker heavy site
2. Note blocking taking place
3. Turn off blocking for this site
4. Use the app logs and/or javascript console to observe no blocking
5. Turn on blocking for this site
6. Note blocking taking place

Test 2:
1. Visit a tracker heavy site
2. Note blocking taking place
3. Turn off blocking for this site
4. Ideally by following a link on that site, navigate to another site
5. Note blocking taking place

Test 3:
1. Visit a tracker heavy site
2. Note blocking taking place
3. Visit settings and disable blocking
4. Return to website
5. Use the app logs and/or javascript console to observe no blocking
6. Visit settings and enable blocking
7. Return to website
8. Note blocking place

###### Reviewer Checklist:


###### PR DRI Checklist:
